libp2p/transports/wstransport.nim

```diff
@@ -63,13 +63,17 @@ template mapExceptions(body: untyped) =
   try:
     body
   except AsyncStreamIncompleteError:
-    raise newLPStreamEOFError()
+    raise newLPStreamIncompleteError()
+  except AsyncStreamLimitError:
+    raise newLPStreamLimitError()
   except AsyncStreamUseClosedError:
     raise newLPStreamEOFError()
   except WSClosedError:
     raise newLPStreamEOFError()
-  except AsyncStreamLimitError:
-    raise newLPStreamLimitError()
+  except WebSocketError:
+    raise newLPStreamEOFError()
+  except CatchableError:
+    raise newLPStreamEOFError()
```

Comment:
If they all raise the same error, why do we need an individual `except` for each one instead of only one `except CatchableError`?

Reply:
`CatchableError` should eventually go away, once `{.async: (raises).}` annotations are applied to `nim-websock`.